bokehjs/test/test_common/test_selection_manager.coffee

```diff
@@ -77,9 +77,6 @@ describe "SelectionManager", ->
 
   sm = new SelectionManager()
 
-  it "should have a reference to an empty selection", ->
-    expect(sm.empty).to.deep.equal empty_selection
```

Comment:
It ocurrs to me that perhaps we're talking past each other because we see the problem differently. 

My understanding of your concern - **please correct me** - is the build up of unhelpful tests that cause a maintenance burden without adding value. 

My concern is that we don't have sufficient tests or a culture of writing tests. The result is frequent regressions, high manual testing burden, and a very high barrier for new contributors (because you can't know without lots experience whether you might be breaking something). For me, I am ok with building up some unhelpful tests if overall we have can improve against the goals of regressions, manual testing, test-driven development (rather than experience-driven) (sorry to overload a term).

I know that lots of unhelpful or flaky tests can erode confidence also, but tests are the only way I know to improve the goals I've mentioned.

I'd be really interested to know if you think I've characterized your concerns correctly, and for you to elaborate on those concerns. We could also move this comment to a more appropriate issue.